BabyPatterns/DispatchVC.swift

```diff
@@ -199,7 +199,11 @@ extension DispatchVC: WCSessionDelegate {
         print("session did deactivate")
     }
 
-    func session(_ session: WCSession, didReceiveMessageData messageData: Data) {
+    func session(_ session: WCSession, didReceiveMessageData messageData: Data, replyHandler: @escaping (Data) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/BabyPatterns/DispatchVC.swift#L202"></span>Line should be 120 characters or less: currently 121 characters<br />
<code>line_length</code> <code>DispatchVC.swift:202</code></td>
    </tr>
  </tbody>
</table>